tyssue/geometry/sheet_geometry.py

```diff
@@ -330,8 +348,47 @@ def scale(eptm, scale, coords):
         eptm.settings["abc"] = [u * scale for u in eptm.settings["abc"]]
 
 
+class EllipsoidLameGeometry(ClosedSheetGeometry):
+    """
+    Sphere surrounding the sheet.
+    Sphere compress the tissue at its extremity
```

Comment:
Here we have also weighted perimeters (in addition to the sphere radius) maybe it would be good to explicit this in the docstring (like: in this geometry, `eptm.vert_df["delta_rho"]` is computed as the difference between the vertex radius in a spherical frame of reference and `eptm.settings['barrier_radius']`) so that we know what happens.